lib/awful/menu.lua

```diff
@@ -669,7 +698,13 @@ function menu.entry(parent, args) -- luacheck: no unused args
         layout:set_right(submenu)
     end
 
+    local enabled = args.enabled == nil and true or args.enabled
+    if not enabled then
+        layout:set_opacity(args.theme.opacity_disabled)
+    end
```

Comment:
We should have a setter defined for this behavior. The user could alter the `enable` value latter and this wouldn't update the opacity.